portfolio/src/main/webapp/script.js

```diff
@@ -30,11 +30,17 @@ function getRandomGreeting() {
   });
 }
 
-/** Creates an <li> element containing text. */
+// Creates an <li> element containing text.
 function createListElement(text) {
   const liElement = document.createElement('li');
   liElement.innerText = text;
   return liElement;
 }
 
+// Creates a map and adds it to the page.
+function initMap() {
+  const map = new google.maps.Map(
+      document.getElementById('map'),
+      {center: {lat: 39.412, lng: -76.775}, zoom: 16, mapTypeId: google.maps.MapTypeId.ROADMAP});
```

Comment:
Might be worth it to add a comment to convey what these coordinates are for in English words.

Reply:
Added the comment (it's Baltimore, MD)

Reply:
nit: Another good practice would be use constants for these literal values. This may also help abstracting any bits later on if necessary.

Reply:
yes, just fixed that!